sdk/lib/opentelemetry/sdk/trace/span.rb

```diff
@@ -118,6 +118,37 @@ def add_attributes(attributes)
           self
         end
 
+        # Add a link to a {Span}.
+        #
+        # Adding links at span creation using the `links` option is preferred
+        # to calling add_link later, because head sampling decisions can only
+        # consider information present during span creation.
+        #
+        # Example>
```

Comment:
Same suggestion as above:
```suggestion
        # Example:
```